languagetool-language-modules/fr/src/main/resources/org/languagetool/rules/fr/grammar.xml

```diff
@@ -117793,8 +117793,12 @@ Foundation, Inc., 51 Franklin St, Fifth Floor, Boston, MA 02110-1301, USA
                     <token>et</token>
                     <token postag="D.*" postag_regexp="yes"/>
                     <token postag="N.*" postag_regexp="yes"/>
+                    <token postag="P.*" postag_regexp="yes" min="0" max="1"/>
+                    <token postag="D.*" postag_regexp="yes" min="0" max="1"/>
+                    <token postag="N.*" postag_regexp="yes" min="0" max="1"/>
                     <token>diffèrent</token>
                     <example>Des temps verbaux différents dont l’emploi et la conjugaison diffèrent.</example>
+                    <example>La raison de cette faute courante vient du fait que l’orthographe et la prononciation de cette expression diffèrent grandement : une liaison entre la lettre « n » et la voyelle « a » est, en effet, prononcée à l’oral, donnant le sentiment que bon pourrait s’écrire bonne.</example>
```

Comment:
nitpick (non-blocking) : For readability, maybe go for a shorter version (but still FP-ing ;) ) like _L’orthographe et la prononciation de cette expression diffèrent grandement : la liaison est faite._ 🥜